fix zscaler 406 keyerror message (#17614)
demisto/content: Packs/Zscaler/ReleaseNotes/1_1_17.md

#### Integrations
##### Zscaler Internet Access

> - Fixed an issue with ***url*** command, which returned a wrong error when receiving 406 error status code.

Reviewer: ```suggestion
- Fixed an issue in the ***url*** command where a non indicative error was returned on error status code 406.
```
